shiny/examples/include_css/app.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+from shiny import *
+
+css_file = os.path.join(os.path.dirname(__file__), "css/styles.css")
```

Comment:
```suggestion
from pathlib import Path

css_file = Path(__file__).parent / "css" / "styles.css"
```

(Note that you should edit this locally because isort will reorganize the imports on save.)